src/localize/languages/pt_br.json

```
@@ -22,13 +22,13 @@
       "live_provider": "Provedor de visualização ao vivo para esta câmera",
       "live_providers": {
         "auto": "Automatico",
-        "frigate": "Frigate",
+        "ha": "Home Assistant (ou seja HLS, LL-HLS, WebRTC nativo)",
         "frigate-jsmpeg": "Frigate JSMpeg",
-        "webrtc": "WebRTC"
+        "webrtc-card": "Cartão WebRTC (ou seja Cartão WebRTC da AlexxIT)"
```

```suggestion
        "webrtc-card": "Cartão WebRTC (de @AlexxIT)"
```

I think there is no point to capitalize the word "Card"/"Cartão".

@LeandroIssa I don't know if we are used to translate "card" or not. 

@dermotduffy this is not a 1:1 translation, but perhaps it's better? English 1:1 would be:

```json
        "webrtc-card": "WebRTC Card (from @AlexxIT)"
```

No matter what, "da" should be changed to "de" (neutral) or "do" (male), because "da" is female.